src/main/fc/runtime_config.h

```diff
@@ -109,6 +109,19 @@ extern uint8_t stateFlags;
 uint16_t enableFlightMode(flightModeFlags_e mask);
 uint16_t disableFlightMode(flightModeFlags_e mask);
 
+typedef struct {
+    uint32_t timestamp;
+    uint16_t year;
+    uint8_t month;
+    uint8_t day;
+    uint8_t hour;
+    uint8_t minute;
+    uint8_t second;
+    uint8_t rssi;
```

Comment:
I think it would be better to store the time and RSSI separately, and use separate MSP commands to transmit them. The time MSP command is only used for resynchronisation, so it's probably enough to send it once a minute or so, whereas the RSSI MSP command should be sent as often as possible, which will be possible at a faster rate if the frame is shorter without the time.

Reply:
Agreed, RSSI should be separate.